netlify/netlify-plugin-nextjs
# fix: include css files in handler functions

### Summary

Includes css files in generated handler functions

### Test plan

1. Visit the Deploy Preview [here](https://ep-css-isr-repro.netlify.app/isr)
2. Open the developer tools, expand the `<head>` tag
3. Confirm that a `<style>` tag is present

Site showing broken behaviour [here](https://nextjs-criticalcss-isr.netlify.app/isr), where the `<style>` tag is missing

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Fixes #1268

![doggo](https://user-images.githubusercontent.com/5655473/178525071-9d10702d-4323-44b2-8847-bc79da9e35ee.jpeg)

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
